### PR TITLE
[docs] Add documentation for LMS_USER_ID usage

### PR DESCRIPTION
**Pull request by Tj-Tracy**

### CODE REVIEW

**Review by oliviaruizknott — APPROVED** [review]
I’ve not been following this work, but this looks good to me. It was easy to follow and didn’t leave any glaring questions.

File: docs/lms_user_id.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#. If a user logs into the system, and navigates to their record page, if a user does not already exist for them in the credentials service, one will be created and the **LMS_USER_ID** will be populated.
+
+Backpopulating Pre-existing users
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Although new users interacting with the system will automatically have this column populated, users who already exist in the system will not. For those users, a `management command`_ exists to backpopulate the data. This command requires an S3 bucket to dump information into. The command also allows for stepped completion, to facilitate manually verifying information before completing the process.
+
+Usage
+~~~~~
+Although credentials now supports **LMS_USER_ID**, it is not currently being widely used within the service. If the data is missing from a user, there should be no issues for general operation of the service, with one exception. If you using the service to automatically send emails to users who have completed programs, it is `required for that user to have the **LMS_USER_ID**`_.
```
Comment (justinhynes) [inline docs/lms_user_id.rst:15]: This isn't true. This is one of the things we learned too late.

We require `lms_user_id` when sending emails to learners through ACE using the _Braze_ channel. However, Credentials is using the _django_email_ channel, and doesn't require the `lms_user_id` (just the email).

**Review by justinhynes — CHANGES_REQUESTED** [review]
I have one tiny change requested to make sure the information is valid. Thanks, TJ!

File: docs/lms_user_id.rst (hunk 1 above)
Comment (oliviaruizknott) [reply docs/lms_user_id.rst:15]: Geez! Glad you also reviewed this, Justin. 😅

Comment (Tj-Tracy) [reply docs/lms_user_id.rst]: good call. deleting it.

**Review by justinhynes — APPROVED** [review]

**Pull request merged**